spyderlib/widgets/sourcecode/syntaxhighlighters.py

```diff
@@ -190,7 +195,35 @@ def set_color_scheme(self, color_scheme):
 
     def highlightBlock(self, text):
         raise NotImplementedError
-            
+
+    def highlight_spaces(self, text, offset=0):
+        """
+        Make blank space less apparent by setting the foreground alpha.
+        This only has an effect when 'Show blank space' is turned on.
+        Derived classes could call this function at the end of
+        highlightBlock().
+        """
+        show_blanks = self.document().defaultTextOption().flags() & QTextOption.ShowTabsAndSpaces

```

Comment:
This line is probably longer than 79?

I am actually flexible on that PEP8 rule but I think @ccordoba12 is more serious about it :-)